libandroid-navigation-ui/javadoc.gradle

```diff
@@ -7,11 +7,9 @@ android.libraryVariants.all { variant ->
     source = variant.sourceSets.collect { it.java.sourceFiles }.inject { m, i -> m + i }
     classpath += variant.javaCompile.classpath
     classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-    options.windowTitle("Mapbox Android Navigation UI SDK $VERSION_NAME Reference")
-    options.docTitle("Mapbox Android Navigation UI SDK $VERSION_NAME")
-    options.header("Mapbox Android Navigation UI SDK $VERSION_NAME Reference")
-    options.bottom("&copy; 2019 Mapbox. All rights reserved.")
```

Comment:
Should this copyright notice also be deleted/kept?
( I am not a lawyer )

It might be silly to delay this PR, just because of some copyright notices;
So maybe it's better to check that in a separate issue/PR.

Reply:
Also not a lawyer, but the copyright seems to be wrong anyway, as the repo is under MIT, so I think it should be alright to do this, but maybe link to the license file or something like that (if it would be hosted on some webpage, which I don't think it is right now?). Either way, we are currently testing and reviewing this PR on our end, so it's not blocked due to that, sorry if there isn't a lot of feedback right now I hope we have some by the end of next week.

Reply:
That said, please feel free to also review the code @frankkienl - my plan would be to merge this ASAP (which is more like 1-2 weeks or so).